### PR TITLE
Issue43 - implement list command with --tenant and --tenant_slug (also prettier format the code)

### PR DESCRIPTION
**Pull request by elv-wayne**
Implemented 2 versions which should return the same results:

`./elv-live list --tenant iten2Fd7H46YFSzP4UaBDhiQEUBHTnEt`
`./elv-live list --tenant_slug sony-live`

Result:
 ```
Network: demov3
.:
  source: >-
    hq__Cg96m7c94PL922s33dpfv87hsdsiJaoPaFF1z7CtcFjhm7Me5PEnZkFStkZx12eeJKvLMNRCcP
asset_type: primary
collections: {}
display_title: SONY Live
images: {}
info:
  copyright: ''
  logo: null
  name: SONY Live
  privacy_policy: ''
  privacy_policy_html: null
  talent: {}
  terms: ''
  terms_html: null
marketplaces:
  spiderman-marketplace:
    .:
      source: >-
        hq__75NZUZSudYQD5XTM8Fb4svvVjfg8Qj52BpRmX5MbbKchbuNrw55TtsWVW3dZMCVNnsBUwZd7eb
    asset_type: primary
    display_title: Spiderman Marketplace
    images: {}
    info:
```

TODO later we want to make marketplaces and sites more concise (just an array of slugs maybe?), and then add --marketplace and --site

### CODE REVIEW

**Review by elv-michelle — APPROVED** [review]
Serban should review too but this is great from my perspective.

File: src/EluvioLive.js [hunk 1]
```diff
@@ -64,84 +57,106 @@ class EluvioLive {
...
+    tenantId,
+    libraryId,
+    objectId,
+    eventId,
+    marketplaceId,
+    cauth,
+    mintHelper,
+  }) {
+    const abiNft = fs.readFileSync(
+      path.resolve(__dirname, "../contracts/v3/ElvTradable.abi")
```
Comment (elv-serban) [inline src/EluvioLive.js:70]: I think this has to be ElvTradableLocal - can you pls try?  Can be a separate commit

**Review by elv-serban — APPROVED** [review]
Looks good!

Did you add a linter for the style.  Or used a linter command that we could add in package.json?  (separate commit)

**Comment by elv-wayne** [comment]
I used "Prettier" within VSCode's format command: Shift + Option + F. (2 spaces for tabs).
I'll see if I can make it automatic.
